### PR TITLE
MANTA-4959 Don't use Math.log10() because it's not available in old versions of node

### PR DESCRIPTION
**Pull request by khalfella**

### CODE REVIEW

**Comment by joshwilsdon** [comment]
Out of curiosity, what's using node 0.10 that can't be updated?

**Comment by khalfella** [comment]
> Out of curiosity, what's using node 0.10 that can't be updated?

**Comment by khalfella** [comment]
> Out of curiosity, what's using node 0.10 that can't be updated?
Muskie for mantav1 uses node v0.10

**Comment by joshwilsdon** [comment]
> Muskie for mantav1 uses node v0.10.

What are we adding to muskie for mantav1 that requires the new artedi?

**Comment by khalfella** [comment]
We are updating muskie to take advantage of the new histogram implementation. See [MANTA-4388](https://jira.joyent.us/browse/MANTA-4388).

File: lib/buckets.js [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
  *
- * Copyright (c) 2018, Joyent, Inc.
+ * Copyright (c) 2020, Joyent, Inc.
  */
 
 var assert = require('assert-plus');
 
+// Math.log10() was introduced in ES6. Old versions of node don't have it.
+function log10(x) {
+    return (Math.log(x) / Math.log(10));
```
[the remark this answers is not in the document]
Comment (khalfella) [reply lib/buckets.js]: Good idea. It seems like the difference is due to rounding float numbers. I updated the code to use the polyfill you mentioned above.

**Pull request merged**